test/SerilogTracing.Tests/LoggerActivityTests.cs

```diff
@@ -29,13 +29,33 @@ public void ExpectedCompletionLevelIsUsed(LogEventLevel initialLevel, LogEventLe
         activity.ActivityTraceFlags |= ActivityTraceFlags.Recorded;
         activity.Start();
 
-        var loggerActivity = new LoggerActivity(logger, initialLevel, activity, MessageTemplate.Empty, []);
+        var loggerActivity = new LoggerActivity(logger, initialLevel, activity, default, MessageTemplate.Empty, []);
         loggerActivity.Complete(completionLevel);
 
         var span = sink.SingleEvent;
         Assert.Equal(expectedLevel, span.Level);
         Assert.Equal(expectedStatusCode, activity.Status);
     }
+    
+    [Fact]
+    public void ExpectedKindIsUsed()
+    {
+        var sink = new CollectingSink();
+
+        var logger = new LoggerConfiguration()
+            .WriteTo.Sink(sink)
+            .CreateLogger();
+
+        using var activity = Some.Activity();
+        activity.ActivityTraceFlags |= ActivityTraceFlags.Recorded;
+        activity.Start();
+
+        var loggerActivity = new LoggerActivity(logger, LogEventLevel.Information, activity, ActivityKind.Producer, MessageTemplate.Empty, []);
+        loggerActivity.Complete();
+
+        var span = sink.SingleEvent;
+        Assert.Equal(ActivityKind.Producer.ToString(), span.Properties["SpanKind"].ToString());
```

Comment:
Could use `Constants.SpanKindPropertyName` here

Reply:
I'd be happy to do that. I usually just try use constant values directly in tests to catch if they're renamed for any reason.

Reply:
Makes sense; let's keep it as-is 👍